hack/functests.sh

```diff
@@ -25,5 +25,6 @@ source hack/config.sh
 if [[ ${TARGET} == openshift* ]]; then
     oc=${kubectl}
 fi
+KUBECONFIG=${KUBECONFIG:-$(cluster/kubeconfig.sh)}
 export KUBEVIRT_PROVIDER=$KUBEVIRT_PROVIDER
-${TESTS_OUT_DIR}/tests.test -kubeconfig=$(cluster/kubeconfig.sh) ${FUNC_TEST_ARGS}
```

Comment:
This is not necessary, with `KUBEVIRT_PROVIDER=external` the `kubeconfig.sh` script will return the `KUBECONFIG` env var

Reply:
as discussed offline, KUBEVIRT_PROVIDER=external does not support cli.sh command which is needed here. 
+ kubeconfig.sh is not dependant of KUBEVIRT_PROVIDER variable.